### PR TITLE
Add missing cli commands

### PR DESCRIPTION
**Pull request by paigecalvert**
Previews:

[api](https://deploy-preview-1467--replicated-docs-upgrade.netlify.app/reference/replicated-cli-api)
[cluster versions](https://deploy-preview-1467--replicated-docs-upgrade.netlify.app/reference/replicated-cli-cluster-versions)
[login](https://deploy-preview-1467--replicated-docs-upgrade.netlify.app/reference/replicated-cli-login)
[logout](https://deploy-preview-1467--replicated-docs-upgrade.netlify.app/reference/replicated-cli-logout)
[release compatibility](https://deploy-preview-1467--replicated-docs-upgrade.netlify.app/reference/replicated-cli-release-compatibility)
[release inspect](https://deploy-preview-1467--replicated-docs-upgrade.netlify.app/reference/replicated-cli-release-inspect)

### CODE REVIEW

File: docs/reference/replicated-cli-cluster-ls.md [hunk 1]
```diff
@@ -49,15 +49,6 @@ replicated cluster ls [flags]
 ## Example
 
 ```bash
-replicated channel ls
-```
```
Comment (paigecalvert) [inline docs/reference/replicated-cli-cluster-ls.md:53]: looks like this had been copied from `channel ls` and never changed

File: docs/reference/replicated-cli-cluster-versions.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Example
+
+```bash
+replicated cluster versions --distribution openshift
+
+Supported Kubernetes distributions and versions are:
+DISTRIBUTION: openshift
+• VERSIONS: 4.13.0-okd
+• INSTANCE TYPES: r1.small, r1.medium, r1.large, r1.xlarge, r1.2xlarge
```
Comment (jdewinne) [inline docs/reference/replicated-cli-cluster-versions.md:41]:
```suggestion
• INSTANCE TYPES: r1.medium, r1.large, r1.xlarge, r1.2xlarge
```

Comment (jdewinne) [reply docs/reference/replicated-cli-cluster-versions.md:41]: Openshift has no instance type `r1.small`

**Review by jdewinne — CHANGES_REQUESTED** [review]
One small change regarding the openshift example instance type

**Review by jdewinne — APPROVED** [review]

**Pull request merged**